data/registry/collector-builder.yml

```diff
@@ -14,6 +14,7 @@ urls:
   repo: https://github.com/open-telemetry/opentelemetry-collector/tree/main/cmd/builder
   docs: /docs/collector/custom-collector/
 createdAt: 2023-12-18
+isFirstParty: true
```

Comment:
Apologies, I should have been more specific in the original issue. This flag is not relevant for collector components it refers to instrumentations that are "first party", i.e. provided by the same org that created a certain app or library. 

We could consider to have them for receivers/exporters hosted outside the community repos by orgs that create such modules fur their solution but I don't know if this is what we want them to do va adopting otlp

Reply:
I was unsure about this change. Thanks for the explanation.

I've made the adjustment to your change request, rebased and adjusted the commit message a bit, then did a force push.

Thanks again!

Reply:
Just to clarify, Do you mean that _all_ collector-exporters don't need this change? In other words, this PR should be abandoned?

Reply:
Depends on how we want to use that flag. We had a thirdParty flag in the past that we removed because it created so much confusion. Right now the first party flag is used in the registry to say what I outlined above. I don't know if we water down the value of that if all otel first party components have that label as well